LLama/LLamaEmbedder.cs

```diff
@@ -97,9 +97,13 @@ public async Task<float[]> GetEmbeddings(string text, bool addBos, CancellationT
 
         private float[] GetEmbeddingsArray()
         {
-            var embeddings = NativeApi.llama_get_embeddings_seq(Context.NativeHandle, LLamaSeqId.Zero);
-            if (embeddings.Length == 0)
-                return Array.Empty<float>();
+            var embeddings = NativeApi.llama_get_embeddings(Context.NativeHandle);
```

Comment:
In my own branch for investigating this issue I have rewritten the various get_embeddings methods to return a raw `float*`, instead of a `Span<float>` (just exposing the llama.cpp symbol directly, instead of wrapping it). Can I suggest making the same change in your PR?

Since `null` is a valid value for these functions to return a Span isn't really a suitable wrapper.

Reply:
Changing to float* is a lot of work because you need to change several functions... maybe it is better if you merge this PR and then change it as you like. Pointers are 'unsafe', so I am not sure if that is a good idea, but I trust that you know what you are doing.

Reply:
Sure, I'll have a look at that once this one is merged :)